Data-Visualization-Project.md

```diff
@@ -0,0 +1,83 @@
+
+
+## Hints and Resources : 
+
+### Step by Step instructions: 
+ 	
+D3 Introduction: https://d3js.org where you can learn about Selections , Dynamic properties and Transitions in D3.js.
+
+•In the main javascript function, write a function to handle Histogram ( Bar chart ) - Histogram will show the total number of students who has taken the quiz ( interacted with that topic ) for 15 topics 
+(Here's the Tutorial : https://bost.ocks.org/mike/bar/  where you can learn how to make a bar chart using the D3 JavaScript library. First tutorial teaches how to make a bare-bones version in HTML, then a more complete chart in Scalable Vector Graphics (SVG), and lastly animated transitions between views.) 
+
+•	Create SVG for histogram ( Resource : http://codepen.io/SundeepB/pen/CxveH ) 
+•	Create function for x-axis mapping and add x-axis to histogram SVG
+•	Create function for y-axis mapping and Create bars for histogram to contain rectangles and topic labels.
+•	create the rectangles and topic labels 
+•	create function to update the bars. This will be used by pie-chart
+
+•	write a function to handle pieChart. – pie chart will have three slices – Low , Mid and High to represent scores 
+ (Tutorial - http://zeroviscosity.com/d3-js-step-by-step/step-1-a-basic-pie-chart where you can learn how to make a pie chart, then transitions between views and how to create a legend  )
+•	create svg for pie chart.
+•	create function to draw the arcs of the pie slices - pie slices will be Low , Mid and high 
+•	create a function to compute the pie slice angles.
+•	Draw the pie slices.
+•	create function to update pie-chart. This will be used by histogram.
+
+•	calculate total frequency by segment for all topic.
+•	calculate total frequency by state for all segment.
+
+
+## Result of Data Analysis and what can we infer from Visualization  :
+
+- Initial pie chart shows aggregate classification of all student score on all topic combined into three categories ‘low, mid ,high’ 
+
+- Initial bar chart showing number of students who have interacted on that particular topic 
+
+- Any Selected category from pie chart will update bar chart, showing number of students who have interacted on various topics having score belonging to that particular category .
+Screenshots below shows the mouse hover on 'Mid' slice and 'High' slice of the pie chart respectively and for that particular slice , topic-wise bar charts and number of students. 
+
+![screen shot 2016-05-17 at 5 13 53 pm](https://cloud.githubusercontent.com/assets/8707780/15343597/ecb77c60-1c52-11e6-85ef-8affa8fc37ce.png)
+
+
+![screen shot 2016-05-17 at 5 14 05 pm](https://cloud.githubusercontent.com/assets/8707780/15343605/f0e2f436-1c52-11e6-890d-d1300a943b4f.png)
+
+
+
+- Any selected bar chart’s bar will update pie chart showing classification of all students score on that particular topic into three categories Low , Mid and High. 
```

Comment:
Thank you for your input @theflametrooper ! 
I have made the changes you suggested but I did't remove the space before 'Any' because removing the space was changing the formatting ( instead of circular bulletin points, it was showing '-' if I remove the space )  and it looked unattractive.
Please feel free to let me know any more improvements !